Add section with Pools

Antoninich opened the pull request:
1) A section with a list of pools sorted by the last forged block has been added to the navigation. There is also the number of registered miners on each pool.
![image](https://github.com/signum-network/signum-explorer/assets/57628432/3a8510a1-36a4-4d7f-b18e-717d157bd3f5)

2) Clicking on the pool address opens a page with detailed data on the pool with sections: Transactions and registered miners.
![image](https://github.com/signum-network/signum-explorer/assets/57628432/8d69ab93-2a0c-4d92-abef-fac91c4927e9)

3) By clicking on the number of miners on the Pools homepage, a list of all registered miners opens.

pir8radio commented:
> 1. A section with a list of pools sorted by the last forged block has been added to the navigation. There is also the number of registered miners on each pool.
>    ![image](https://user-images.githubusercontent.com/57628432/282269605-3a8510a1-36a4-4d7f-b18e-717d157bd3f5.png)
> 2. Clicking on the pool address opens a page with detailed data on the pool with sections: Transactions and registered miners.
>    ![image](https://user-images.githubusercontent.com/57628432/282269678-8d69ab93-2a0c-4d92-abef-fac91c4927e9.png)
> 3. By clicking on the number of miners on the Pools homepage, a list of all registered miners opens.

thanks for adding screenshots of the changes.   i like to see what the PR's do in the wild!

Antoninich commented:
Added timestamp on first page Pools and changed the location of the columns. The Blocks column comes first.
![image](https://github.com/signum-network/signum-explorer/assets/57628432/3d5abc49-8193-479b-87e9-e9df5ca47ec2)

frankTheTank72 commented on config/urls.py:
```diff
@@ -1,60 +1,40 @@
     PeerMonitorListView,
     peers_charts_view,
 )
-
-from scan.views.json import (
-    getSNRjson,
-    getStatejson,
-    TopAccountsJson,
-)
-
```
Is this not longer needed ? Did you checked that ?

Antoninich replied:
In urls.py this import not use, so it can be deleted.

Antoninich replied:
> Is this not longer needed ? Did you checked that ?

Sorry. This import has been moved to line 21. Moved to match PEP8.

frankTheTank72 replied:
ah okay , thanks - and without () that will work on line 21 ?

Antoninich replied:
> ah okay , thanks - and without () that will work on line 21 ?

Yes, () is just a way of displaying multiple objects, where each object is on a new line.

pir8radio commented:
@Antoninich  can you modify this to exclude any lines with the same pool address?     only the latest reference to a pool should be listed so this is a true list of pools.    see production explorer https://explorer.notallmine.net/pools/              it just lists every forged block by a pool.   this is due to the trimming being off i believe.   removing duplicates should be a decent patch.
